NEWS.md

```diff
@@ -1,15 +1,76 @@
+* Fix restarting mount units on snapd start
+* Fix call to udev in preseed mode
+* Fix to ensure always setting up the device cgroup for base bare and core24+
+* Fix not copying data from newly set homedirs on revision change
+* Fix leaving behind empty snap home directories after snap is removed (resulting in broken symlink)
+* Fix to avoid using libzstd from host by adding to snapd snap
+* Fix autorefresh to correctly handle forever refresh hold
+* Fix username regex allowed for system-user assertion to not allow '+'
+* Fix incorrect application icon for notification after autorefresh completion
+* Fix to restart mount units when changed
+* Fix to support AppArmor running under incus
+* Fix case of snap-update-ns dropping synthetic mounts due to failure to match  desired mount dependencies
+* Fix parsing of base snap version to enable pre-seeding of Ubuntu Core Desktop
+* Fix packaging and tests for various distributions
+* Add remoteproc interface to allow developers to interact with Remote Processor Framework which enables snaps to load firmware to ARM Cortex microcontrollers
+* Add kernel-control interface to enable controlling the kernel firmware search path
+* Add nfs-mount interface to allow mounting of NFS shares
+* Add ros-opt-data interface to allow snaps to access the host /opt/ros/ paths
+* Add snap-refresh-observe interface that provides refresh-app-awareness clients access to relevant snapd API endpoints
+* steam-support interface: generalize Pressure Vessel root paths and allow access to driver information, features and container versions
+* steam-support interface: make implicit on Ubuntu Core Desktop
+* desktop interface: improved support for Ubuntu Core Desktop and limit autoconnection to implicit slots
+* cups-control interface: make autoconnect depend on presence of cupsd on host to ensure it works on classic systems
+* opengl interface: allow read access to /usr/share/nvidia
+* personal-files interface: extend to support automatic creation of missing parent directories in write paths
+* network-control interface: allow creating /run/resolveconf
+* network-setup-control and network-setup-observe interfaces: allow busctl bind as required for systemd 254+
+* libvirt interface: allow r/w access to /run/libvirt/libvirt-sock-ro and read access to /var/lib/libvirt/dnsmasq/**
+* fwupd interface: allow access to IMPI devices (including locking of device nodes), sysfs attributes needed by amdgpu and the COD capsule update directory
+* uio interface: allow configuring UIO drivers from userspace libraries
+* serial-port interface: add support for NXP Layerscape SoC
+* lxd-support interface: add attribute enable-unconfined-mode to require LXD to opt-in to run unconfined
+* block-devices interface: add support for ZFS volumes
+* system-packages-doc interface: add support for reading jquery and sphinx documentation
+* system-packages-doc interface: workaround to prevent autoconnect failure for snaps using base bare
```

Comment:
Is this supposed to be the system-packages-doc interface? Seems like it could potentially be a typo.